internal/pkg/client/ocisif/ocisif.go

```diff
@@ -14,6 +14,8 @@ import (
 	"strings"
 	"time"
 
+	ociref "github.com/containers/image/v5/docker/reference"
+	ociconfig "github.com/containers/image/v5/pkg/docker/config"
```

Comment:
We definitely want to avoid adding `containers/image` dependencies here. if possible.

This package is `containers/image` own implementation of the docker config file that we are already using `docker/cli` for elsewhere.